src/refactor_nrepl/ns/tracker.clj

```diff
@@ -36,13 +37,16 @@
       ;; corner case - use the mranderson-ized refresh-dirs (for supporting this project's test suite):
       refresh-dirs))
 
+(def default-predicate (every-pred core/source-file?
```

Comment:
`default-file-predicate` or maybe even something like `filter-file-predicate`?  The latter gives some indication that we're using this predicate to filter the files to be processed.